base_product_mass_addition/models/product_product.py

```diff
@@ -26,6 +26,7 @@ class ProductProduct(models.Model):
         domain="[('category_id', '=', quick_uom_category_id)]",
         compute="_compute_quick_uom_id",
         inverse="_inverse_set_process_qty",
+        store=True,
```

Comment:
**issue**: you try to store a field that should be computed on the fly, and not be stored. I guess something is wrong with that design. could you explain that change ? Thanks !

Reply:
This is the issue without ```store=True```
```
tests/test_product_mass_addition.py::TestProductMassAddition::test_quick_line_add
  /odoo/src/odoo/fields.py:808: UserWarning: Field product.product.quick_uom_id in dependency of product.product.quick_uom_category_id should be searchable. This is necessary to determine which records to recompute when uom.uom.category_id is modified. You should either make the field searchable, or simplify the field dependency.
```

It seems there is some kind of dependency race on this.

However I'm not sure there is a need to have the ```quick_uom_category_id``` field since it's not use anywhere in the module and it could be accessed via ```quick_uom_id.category_id```